lib/charts/chart.rb

```diff
@@ -24,8 +24,10 @@ def validate_arguments(data, options)
 
   def array?(options, keys)
     keys.map do |key|
-      if options[key] && !options[key].is_a?(Array)
-        raise ArgumentError, "#{key} not an array"
+      if options[key]
+        unless options[key].is_a?(Array) 
```

Comment:
Since options[key] will either exist or not and it will be what ever it is, you should be able to simply ask if it is an array, if it really must be an array.